consumer/src/test/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/saker/SakerServiceImplTest.kt

```diff
@@ -125,36 +118,21 @@ class SakerServiceImplTest {
 
     @Test
     fun `oppretter ikke generell oppfolgingssak og fjerner generell oppfolgingssak dersom fagsaker inneholder oppfolgingssak`() {
-        val wsSaker: List<WSSak> = listOf(
```

Comment:
Jeg er personlig mer fan av å ha koden inline her, siden jeg da kan se alt relevant informasjon for testen.

Reply:
Jeg har slått sammen koden fra  `SakDataGenerator` i `SakerServiceImplTest`.

Reply:
Men hvorfor skal vi trekke kode som brukes ett enkelt sted ut av filen? Det gjør det jo bare vanskeligere å forstå testen.

Reply:
Jeg bare gjort det å reduser størelse av kode i den filen.